documentation/architecture/README.md

```diff
@@ -81,3 +88,44 @@ The "analytics" component of the application is the parts composed of the analyt
 - [Application Infrastructure Service](../wiki/decisions/adr/2023-07-20-deployment-strategy.md)
 - [Analytics Data Storage](../wiki/decisions/adr/2024-03-19-dashboard-storage.md)
 - [Analytics Dashboard Tool](../wiki/decisions/adr/2024-04-10-dashboard-tool.md)
+
+## Architecture Description
+
+### Overview
+
+Currently a single AWS Project houses all of the environments.
+
+- [Layers](#layers)
+- [Traffic Flow](#traffic-flow)
+  - [Front End Requests](#front-end-requests)
+  - [API Requests](#api-requests)
+  - [File Requests](#file-requests)
+- [SOAP S2S Replacement](#soap-s2s-replacement)
+
+### Layers
+
+AWS Cloudfront and API Gateway are deployed in front of public facing HTTPs services including the Front End and Back End Services. Access to Services is via AWS ALBs mapping to origins hosted on ECS.
+
+Services, including FE NextJS service, BE Python API service, Metabase service, Python Analytics Service, and Simpler NOFOs Python service are all hosted as containers running on ECS clusters. These services are set to auto scale as needed to handle increases in traffic. AWS Parameter Store is used to adjust Environment Variables being passed to the services to control behavior by environment and to support feature flags defaults for the NextJS and Python API services.
+
+Managed AWS Services are used for our data storage layer, including AWS RDS for PostgreSQL, Manged OpenSearch for search index, and AWS S3 for file storage.
+
+### Traffic Flow
+
+#### Front End Requests
+
+The public face of the project allows HTTPS traffic from the Internet to reach the AWS Cloudfront Content Delivery Network (CDN) configured for [simpler.grants.gov](https://simpler.grants.gov), [training.simpler.grants.gov](https://training.simpler.grants.gov), etc. The CDN routes those requests to AWS Application Load Balancers (ALBs). Those ALBs route the requests to the Elastic Container Service (ECS) Clusters and Services that host the Front End code, written in NextJS.
```

Comment:
is it worth noting the CDN (at least in regards to caching) is not enabled in PROD?